Safety/Src/PWM.cpp

```diff
@@ -3,31 +3,34 @@
 #include "PWM.hpp"
 #define MAX_CHANNELS 10
 #define NUM_AVAILABLE_CHANNELS 8
+#define DSHOT_DATA_FRAME_LEN 16
+#define DSHOT_DMA_BUFFER_SIZE 18
```

Comment:
Why does this need to be 18 instead of 16 (considering the frame length is 16). 
See also line 96 
```
//Setting the last two indexes in the buffer to 0 to ensure the last part of the signal is low
dmaBuffer[16] = 0;
dmaBuffer[17] = 0;
```

Reply:
This is because the first 16 bits is the actual dshot frame but we are reserving 2 extra bits to ensure the signal is LOW after the frame has passed. I'm actually not too sure if just 1 extra bit will do but that is what I have seen in other implementations